releasenotes/notes/feat-polytensor-apply-multi-a3deea9586e6451a.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    Adds a new ``multi`` argument to the :meth:`PolynomialTensor.apply` method which
+    allows handling of numpy routines which return more than one array.
+    The same argument also gets exposed by :meth:`ElectronicIntegrals.apply`.
```

Comment:
These do not end up as links - do they not need a leading `.` for this to work e.g ``:meth:`.PolynomialTensor.apply` ``